src/main/flight/mixer.c

```diff
@@ -929,9 +934,16 @@ FAST_CODE_NOINLINE void mixTable(timeUs_t currentTimeUs, uint8_t vbatPidCompensa
     } else {
         if (airmodeEnabled || throttle > 0.5f) {  // Only automatically adjust throttle when airmode enabled. Airmode logic is always active on high throttle
             throttle = constrainf(throttle, -motorMixMin, 1.0f - motorMixMax);
+#ifdef USE_AIRMODE_LPF
+            airmodeThrottleChange = constrainf(loggingThrottle, -motorMixMin, 1.0f - motorMixMax) - loggingThrottle;
```

Comment:
If `loggingThrottle` is used here then it should be renamed, as it's obviously not used exclusively for logging any more.

Reply:
Also the goal of `loggingThrottle` is to show the final throttle value used in the motor mixing. Thus allowing comparison to the commanded vs. final throttle as modified by other flight control systems. So `loggingThrottle` should include the offset generated by `pidGetAirmodeThrottleOffset()`.

Reply:
@etracer65, sorry I overlooked this one. I'm confused since loggingThrottle does not include the offset airmode generally adds to throttle. It seems we should either include all airmode generated offsets or none. Which do you think would be more appropriate?